src/server/network/protocol/protocolgame.cpp

```diff
@@ -1360,10 +1360,7 @@ void ProtocolGame::parseHotkeyEquip(NetworkMessage &msg) {
 	if (!player) {
 		return;
 	}
-
-	uint16_t itemId = msg.get<uint16_t>();
-	uint8_t tier = msg.get<uint8_t>();
-	g_game().playerEquipItem(player->getID(), itemId, Item::items[itemId].upgradeClassification > 0, tier);
+	player->sendCancelMessage("You cannot equip items by using a hotkey.");
```

Comment:
Like this? These modifications don't seem to make sense. Won't this make the functionality stop working?

Reply:
I am sorry but I didn't wanted to pull this to your repository. Totally my bad